themes/bootstrap3/templates/RecordTab/search2collectionlist.phtml

```diff
@@ -0,0 +1 @@
+collectionlist.phtml
```

Comment:
Is there supposed to be more content here? Unless this is using a syntax I'm not familiar with, it seems incomplete.

Reply:
It's a symlink between `collectionlist.phtml` > `search2collectionlist.phtml` to avoid duplicating the template. Maybe `RecordTab\Search2CollectionList` could be somehow set to render `collectionlist.phtml` instead.

Reply:
Try `<?=$this->render('RecordTab/collectionlist.phtml')?>` instead -- that should have the same effect, but in a more platform-independent way.

Reply:
Yes, better this way.